components/menu/stories/menu.stories.js

```diff
@@ -1,4 +1,6 @@
-// Import the component markup template
+// import isChromatic from "chromatic/isChromatic";
+const isChromatic = () => true;
```

Comment:
Ope I think this is a holdover from testing, we want to use the flag from Chromatic right?

Reply:
Yes thank you!  Good catch.